msteams-platform/bots/how-to/authentication/auth-aad-sso-bots.md

```diff
@@ -6,7 +6,7 @@ keywords: token, user token, SSO support for bots
 
 # Single sign-on (SSO) support for bots
 
-Single sign-on authentication in Azure Active Directory (Azure AD) minimizes the number of times users need to enter their login credentials by silently refreshing the authentication token. If users agrees to use your app, they will not have to consent again on another device and will be signed in automatically. The flow is very similar to the [Teams tab SSO support]( ../../../tabs/how-to/authentication/auth-aad-sso.md). The difference is the protocol for how a bot requests tokens and receives responses.
+Single sign-on authentication in Azure Active Directory (Azure AD) minimizes the number of times users need to enter their login credentials by silently refreshing the authentication token. If users agree to use your app, they will not have to consent again on another device and will be signed in automatically. The flow is very similar to the [Teams tab SSO support]( ../../../tabs/how-to/authentication/auth-aad-sso.md). The difference is the protocol for how a bot requests tokens and receives responses.
```

Comment:
Global comment for images:

Please ensure that you follow proper naming convention for images. 

Verify "png" or "PNG"